.github/ISSUE_TEMPLATE/release.yaml

```diff
@@ -0,0 +1,100 @@
+name:  Release
+description: Release/Pre-release of a version.
+labels: [release]
+body:
+- type: dropdown
+  id: release_type
+  attributes:
+    label: Release Type
+    description: which type of release is this release?
+    options:
+    - major release
+    - minor release
+    - patch release
+- type: textarea
+  id: release_version
+  attributes:
+    label: Release Version
+    description: Version of this release (e.g `2.4.0`)
+    placeholder:
+- type: checkboxes
+  id: release_tests
+  attributes:
+    label: "**For all releases** Github Workflow Test Matrix Checkup"
+    description: |
+      We maintain some integration tests with 3rd party components which we need to manually verify and update before cutting any release.
+      An issue exists to automate the following actions: https://github.com/Kong/kubernetes-ingress-controller/issues/1886
+      **Wait for tests to pass before continuing with any release**, if any problems are found hold on the release until patches are provided and then run the tests again.
```

Comment:
This is a combination of outdated (tests happen later now) and not that useful (the issue exists fine on its own), so this part should definitely be gone.

```suggestion
```
The entire description should probably be dropped given the issues with those not rendering on the submitted issue, but only striking the parts that should definitely go for now.